README-template.md

```diff
@@ -1,44 +1,57 @@
 # TU Dresden - Computer Science: Open Source Projects
 
-This repository contains functionality to generate a HTML and markdown site of all projects entered in `projects.yaml`.
+This repository contains functionality to generate a HTML and markdown site of all projects entered in `oss-projects.yaml` and computer science groups in `cs-groups.yaml`.
```

Comment:
This repository contains scripts and data to generate HTML and markdown files presenting open source software projects of the [Faculty of Computer Science](https://cs.tu-dresden.de) at [TU Dresden](https://www.tu-dresden.de). The sources of information are `oss-projects.yaml`, which describes the projects, and `cs-groups.yaml`, which describes the research groups involved.